addons/l10n_br/models/account_fiscal_position.py

```diff
@@ -21,22 +21,17 @@ class AccountFiscalPosition(models.Model):
         string='Interstate Fiscal Position Type',
     )
 
-    @api.model
-    def _get_fiscal_position(self, partner, delivery=None):
-        if not delivery:
-            delivery = partner
+    def _get_fpos_ranking_functions(self, partner):
+        if self.env.company.account_fiscal_country_id.code != "BR" or partner.country_id.code != 'BR':
+            return super()._get_fpos_ranking_functions(partner)
```

Comment:
I see this done in other overrides of this function but don't understand it. It seems to include `l10n_br_fp_type` fiscal positions for international transactions, but that doesn't seem right.

Isn't it ok to include the `l10n_br_fp_type` tuple for international transactions as well?

Reply:
But then we need a type international